app/controllers/admin/users_controller.rb

```diff
@@ -19,8 +19,12 @@ def index
   end
 
   protected
-  def collection
-    @users ||= apply_scopes(end_of_association_chain).with_user_totals.order_by(params[:order_by] || 'coalesce(user_totals.sum, 0) DESC').includes(:user_total).page(params[:page])
+  def collection(format = :html)
```

Comment:
ok, então não precisa mudar nos outros, muda só aqui, conforme for lidando nos outros controllers vcs mudam, na verdade é mais o nome do parametro isso